elooplib/archive.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#  define SET_BINARY_MODE(file)
+
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include "zlib.h"
```

Comment:
Use angle brackets for system libraries

Reply:
is zlib a system library?

Reply:
Yes, it's installed to /usr/include and may require installing extra package.

If you include with regular quotes - it means you ship zlib with your repository or it gets downloaded somewhere locally